float32 angular_speed         # ID of the detected AprilTag
float32 radius
float32 left_speed
float32 right_speed
float32 x            # Relative x position of the marker in the camera frame
float32 y            # Relative y position of the marker in the camera frame          # Relative z position of the marker in the camera frame
float32 theta       # Relative angle (yaw) of the marker in radians
